dev_guide/secrets.adoc

```diff
@@ -225,6 +225,24 @@ applications that need out-of-the-box certificates. It has the same settings as
 the server certificates generated by the administrator tooling for nodes and
 masters.
 
+The service serving certificate secrets are:
+
+----
+default	                          router-metrics-tls
+kube-service-catalog	            controllermanager-ssl
+openshift-ansible-service-broker	asb-tls
+openshift-console	                console-serving-cert
+openshift-infra	                  heapster-certs
+openshift-logging	                prometheus-tls
+openshift-monitoring	            alertmanager-main-tls
+openshift-monitoring	            grafana-tls
+openshift-monitoring            	kube-state-metrics-tls
+openshift-monitoring            	node-exporter-tls
+openshift-monitoring	            prometheus-k8s-tls
+openshift-template-service-broker	apiserver-serving-cert
+openshift-web-console	            webconsole-serving-cert
+----
```

Comment:
no issue for openshift-monitoring secrets
FYI: It is better to have a good format for all the secrets

Reply:
@juzhao Thank you! I reformatted the information as a table:
http://file.rdu.redhat.com/~ahardin/April-15-2020/service-serving-certs/dev_guide/secrets.html#service-serving-certificate-secrets